modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form/renderer/SuccessVariant.es.js

```diff
@@ -78,16 +78,12 @@ export function Page({page: {successPageSettings}}) {
 	const {initialBody, initialTitle} = {
 		initialBody:
 			(successPageSettings.body &&
-				(successPageSettings.body[
-					Liferay.ThemeDisplay.getLanguageId()
-				] ||
+				(successPageSettings.body[editingLanguageId] ||
 					successPageSettings.body[defaultLanguageId])) ||
 			'',
 		initialTitle:
 			(successPageSettings.title &&
-				(successPageSettings.title[
-					Liferay.ThemeDisplay.getLanguageId()
-				] ||
+				(successPageSettings.title[editingLanguageId] ||
 					successPageSettings.title[defaultLanguageId])) ||
```

Comment:
@vngoliveira your change is reverting the fix made for https://liferay.atlassian.net/browse/LPD-12649.

1. Change the language in the url to Spanish adding the “/es/” after the domain: http://localhost:8080/es/
2. Add a new Form
3. With your changes, the success message is now empty

![image](https://github.com/liferay-forms/liferay-portal/assets/34186632/0ce38fa9-9171-4a90-a4fb-2cb0a6e64e09)

Could you please check?